classes/Inbox.php

```diff
@@ -558,6 +558,7 @@ public static function allConvosForUser($system, $user_id): array {
         $convos_data = $system->db_fetch_all($result);
 
         foreach($convos_data as $convo) {
+            $convo['title'] = htmlspecialchars_decode($convo['title'], ENT_QUOTES);
```

Comment:
Because the Inbox page where this gets displayed is React-based this is okay, but we should be extremely careful with this as a general pattern. 

Any desanitization of data that gets displayed in a PHP template or in React UI using `dangerouslySetInnerHtml` can introduce XSS vulnerabilities